### PR TITLE
Initialize module

### PR DESCRIPTION
**Pull request by brandones**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+node_js:
+  - node
+script:
+  - npm run lint
+  - npm test
+  - npm run typescript
+  - npm run build
+  - echo "Folder name in Digital Ocean Spaces - $TRAVIS_COMMIT"
+  - mkdir -p dist/@openmrs/esm-primary-navigation/$TRAVIS_COMMIT
+  - mv dist/*.* dist/@openmrs/esm-primary-navigation/$TRAVIS_COMMIT/
```
Comment (fatmali) [inline .travis.yml]: change `esm-primary-navigation` to reflect the modules name

Comment (brandones) [reply .travis.yml]: Did that, and also removed the "after deploy" step to update import_map. Which I think is the right thing to do?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  },
+  "dependencies": {
+    "@openmrs/esm-api": "^1.0.3",
+    "@openmrs/esm-module-config": "^1.0.0",
+    "@types/react": "^16.9.2",
+    "@types/react-dom": "^16.9.0",
+    "@types/react-router-dom": "^4.3.5",
+    "react": "^16.9.0",
+    "react-dom": "^16.9.0",
+    "rxjs": "^6.5.3"
```
Comment (joeldenning) [inline package.json]: `rxjs` recently got moved out of the import map, in favor of each package bundling it individually into their packages. See related https://github.com/openmrs/openmrs-esm-root-config/pull/30.

As such, we should move `rxjs` to devDependencies.

File: src/declarations.d.tsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// This entire file will be deleted once we have typescript definitions
```
Comment (joeldenning) [inline src/declarations.d.tsx]: The typescript definitions for esm-api are published to npm now -- we can delete this comment.

File: src/declarations.d.tsx [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+// for @openmrs/esm-api published to npm
+declare type Observable<T> = import("rxjs").Observable<T>;
```
Comment (joeldenning) [inline src/declarations.d.tsx]: Why is this needed? Doesn't the `rxjs` package already have definitions for observables?

Comment (brandones) [reply src/declarations.d.tsx]: Copy-paste detritus

**Comment by brandones** [comment]
Thanks @joeldenning and @fatmali !
